app/controllers/dashboard_controller.rb

```diff
@@ -1,5 +1,7 @@
 class DashboardController < ApplicationController
   def index
     @merchant = Merchant.find(params[:merchant_id])
+    @customers = @merchant.top_customers
```

Comment:
I'm wondering if we could refactor and find a way to just pass one instance variable to the view, such as calling something like @merchant.top_customers in the view instead of @customers.  The same for @merchant.ready_items.

Reply:
I'm glad you mentioned that about the instance variables.  I was wondering if I should have done that as well.  I think I should be able to do it how you suggested to make it skinnier.

Reply:
Yeah this was a great catch.  Thanks for pointing this out.  I've moved them and tests are still happy of course.